astropy_timeseries/tests/test_binned.py

```diff
@@ -3,17 +3,11 @@
 
 from astropy import units as u
 from astropy.table import Table
-from astropy.time import Time
+from astropy.time import Time, TimeDelta
 
 from ..binned import BinnedTimeSeries
 
 
-INPUT_TIME = Time(['2016-03-22T12:30:31',
-                   '2015-01-21T12:30:32',
-                   '2016-03-22T12:30:40'])
-PLAIN_TABLE = Table([[1, 2, 11], [3, 4, 1], [1, 1, 1]], names=['a', 'b', 'c'])
-
-
```

Comment:
These were not used, would you like a version of this for the `binned` like the `sampled` ones?

Reply:
No I think it's fine as-is